src/common/bots/Bot.cpp

```diff
@@ -0,0 +1,110 @@
+#include "Bot.hpp"
+
+Bot::Bot() {};
+
+std::pair<InputDef::ReliableInput, InputDef::UnreliableInput>
+    Bot::vectorToNearestMoveTransform(sf::Vector2f v) {
```

Comment:
I would move this to `common/util/Util.hpp`
We have the opposite function there that converts inputs into a vector (used by the server)

Reply:
I saw that, but making it InputDef aware felt ugly

Reply:
InputDef is just input constants - stuff that's related to inputs. nbd
the real concern is the dependency on networking but we can fix that later

Reply:
done